task/topic.go

```diff
@@ -66,21 +70,21 @@ func processTopic(ctx context.Context, serviceAuthToken string, topicClient topi
 	// Check if the topic has already been processed
 	if _, exists := processedTopics[topicID]; exists {
 		err := errors.New("topic already processed")
-		log.Error(ctx, "Skipping already processed topic", err, log.Data{
+		log.Info(ctx, "Skipping already processed topic", log.Data{
 			"topic_id": topicID,
 			"depth":    depth,
+			"error":    err.Error(),
```

Comment:
These are being processed as errors now so should stay as `log.Error(…)`

Reply:
Not this particular error actually, this is just ensuring I don't have duplicate items in the map